webapps/frontend/ui/admin/client/scripts/camunda-admin-bootstrap.js

```diff
@@ -175,9 +177,20 @@ define('camunda-admin-bootstrap', function() {
           // not have been defined yet. Placing a new require call here will put
           // the bootstrapping of the angular app at the end of the queue
```

Comment:
So the comment is about the plugin dependencies that are already registered & loaded before these lines.

Reply:
This comment concerns the requireJS plugins registered some lines above via `requirejs#config`. After initializing the web app, no `require` and `define` calls should happen since we assume all plugins have been registered already. We are not introducing new behavior here but revert what was there already for quite some years:
https://github.com/camunda/camunda-bpm-platform/blob/3b4fbf9731564e23fba74b542690c1a597546406/webapps/frontend/ui/tasklist/client/scripts/camunda-tasklist-bootstrap.js#L185-L186